docs/usage_guide/advanced_usage/modin_logging.rst

```diff
@@ -0,0 +1,41 @@
+Logging and Tracing with Modin
+==============================
+
+Modin logging and tracing allows users to gain observability into their queries by logging internal Modin API calls, partition metadata, 
```

Comment:
Isn't this the exact same paragraph as above? Maybe change the above one to:
Modin also includes the ability to log internal API calls and metadata, to offer users greater insight into their queries. Please see our docs ... for more.

Reply:
I think it's fine for the descriptions to be the same here. The descriptions are the same for XGBoost and Spreadsheet API as well. It's a fairly short description, and I think that each line in the summary is important (what logging logs, default disabled, and that it is a local file).